br/pkg/task/backup_ebs.go

```diff
@@ -371,7 +373,7 @@ func waitUntilAllScheduleStopped(ctx context.Context, cfg Config, allStores []*m
 				return errors.Trace(err)
 			}
 
-			storeLeaderRegions := make([]*metapb.Region, 0, 100)
+			storeLeaderRegions := storeLeaderRegionsFlat[i*100 : 0 : (i+1)*100]
```

Comment:
Is this the same 100? does it need to be kept in sync with the other 100 on line 357? If so, maybe put it in a variable.